service/dap/server.go

```diff
@@ -1680,11 +1685,19 @@ func (s *Session) onContinueRequest(request *dap.ContinueRequest, allowNextState
 	s.runUntilStopAndNotify(api.Continue, allowNextStateChange)
 }
 
-func fnName(loc *proc.Location) string {
+func fnName(loc *proc.Location, hidePackagePaths bool) string {
 	if loc.Fn == nil {
 		return "???"
 	}
-	return loc.Fn.Name
+	name := loc.Fn.Name
+	if idx := strings.LastIndex(name, "/"); idx != -1 {
```

Comment:
This doesn't always work with functions that have type parameters (the last slash could be inside the type parameter list), the methods of proc.(*Functions) should be used instead.

Reply:
Should be fixed in ca8e87194a50, please have a look. Do you think it's worth adding a fixture that allows us to test this? (I wouldn't mind if the answer is "no" 😄)

Reply:
https://go.dev/play/p/-hZGE17iZxT

The symbol name for the instance of typeparametric function that is called is: `main.typeparametric[go.shape.struct { example.com/blah/reader.x int }]`

Reply:
I see, thanks. OK, I thought it was worth adding tests for this, so I added simple tests for just the `fnName` function: 99483f9ff687